engine/utils/collections/HeapArray.h

```diff
@@ -267,7 +267,7 @@ class MHArray
      * @param index the index of the element we want to check
      * @return `true` if this element has been set, `false` if it has not
      */
-    bool Active(size_t index)
+    bool Active(size_t index) const
```

Comment:
Unrelated to current PR, but generally not as informative as calling this `IsActive`

Reply:
Changed.